gitpod-io/gitpod
# Fix process termination and add additional mknod capability

*How to test:*
- Open https://aledbf-killdocker.staging.gitpod-dev.com/
- Start a prebuild https://aledbf-killdocker.staging.gitpod-dev.com/#prebuild/https://github.com/shaal/gitpod-bug-repro/tree/bug/docker-pull
- ws-daemon should not contain errors restauring snapshot
- gitpod UI should look like:

![Screenshot from 2021-04-01 13-20-48](https://user-images.githubusercontent.com/161571/113324218-3f30cf80-92ed-11eb-83f3-1bd9b6d282f9.png)
![Screenshot from 2021-04-01 13-21-47](https://user-images.githubusercontent.com/161571/113324227-422bc000-92ed-11eb-9456-ead8f51ccda7.png)


fixes #3671
